toolchain/get.go

```diff
@@ -15,21 +17,19 @@ import (
 func CloneOrUpdate(path string, update bool) (*Info, error) {
 	path = filepath.Clean(path)
 
-	dir, err := Dir(path)
-	if err != nil {
-		return nil, err
-	}
+	dir := filepath.SplitList(srclib.Path)[0]
```

Comment:
This is incorrect. I think this should still use the `Dir` function in this package. Otherwise, instead of installing the toolchains in `~/.srclib/` (which is where they are supposed to be), this will install them into `~` (the parent of `$SRCLIBPATH`).